### DISCUSSION TITLE
emails stuck in outbox

### DISCUSSION THREAD

**Discussion by e8vww** [Extended-Thunder/send-later#426] [Q&A]
Sometimes when I use Send Later the emails end up unsent, stuck in the local outbox.

I tried searching the help articles but found no results for this specific problem. Thanks for your help.

thunderbird 91.6.1, send later 9.0.9

**Comment by jph-sendlater** [comment]
Does `File` > `Send Unsent Messages` work to manually send the messages from the outbox?
